Dockerfile-backup-driver

```diff
@@ -12,13 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:focal
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends ca-certificates wget bzip2 && \
-    apt-get remove -y wget bzip2 && \
-    rm -rf /var/lib/apt/lists/*
-RUN mkdir /vddkLibs
```

Comment:
Is there any specific reason why remove this line? Did you confirm that `COPY` command would create the directory if not exist?

Reply:
Confirmed that COPY command would create the directory if not exist.